crates/services/block_aggregator_api/proto/api.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package blockaggregator;
+
+message BlockHeightRequest {}
+
+message BlockHeightResponse {
+  uint32 height = 1;
+}
+
+message BlockRangeRequest {
+  uint32 start = 1;
+  uint32 end = 2;
+}
+
+message Block {
```

Comment:
The idea it to convert `FuelBlock` into a protobuf block, where protobuf block has all fields of `FuelBlock`. We don't plan to use any other serialization, so it is not clear to me what is `data` field here.